docs/guides/writing-a-feature-app.md

```diff
@@ -88,9 +87,50 @@ properties:
 1. `idSpecifier` — An optional [ID specifier][idspecifier] that distinguishes
    the Feature App instance from other Feature App instances with the same ID.
 
-The return value of the `create` method can vary depending on the integration
-solution used. Assuming the [`@feature-hub/react` package][react-api] is used, a
-Feature App can be either a **React Feature App** or a **DOM Feature App**.
+The return value of the `create` method can vary depending on how the integrator
+places Feature Apps on a web page. Assuming React is used, a Feature App can be
```

Comment:
I think you went overboard here with the use of the phrase "place ... on a web page". I think is is fine to use it in the introduction phrase but at this point the reader should know we mean by integration. The `create` method is dependent upon the mechanics of the integrator in many ways. only one of which is how it renders...